### PR TITLE
Openstack image username

### PR DESCRIPTION
**Pull request by lwatta**
Fix for enhancement #780 
Default username is still "ubuntu" but user can use --openstack-image-username to set it something different.

### CODE REVIEW

File: perfkitbenchmarker/providers/openstack/os_virtual_machine.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class OpenStackVirtualMachine(virtual_machine.BaseVirtualMachine):
     """Object representing an OpenStack Virtual Machine"""
 
     CLOUD = 'OpenStack'
-    DEFAULT_USERNAME = 'ubuntu'
+    DEFAULT_USERNAME = FLAGS.openstack_image_username
```
Comment (skschneider) [inline perfkitbenchmarker/providers/openstack/os_virtual_machine.py]: DEFAULT_USERNAME is no longer a constant, nor is its name accurate anymore. I suggest removing it and initializing self.user_name from the flag in the __init__ method.

**Comment by skschneider** [comment]
Please rebase the changes in your branch. This pull request is currently showing additional commits to your branch that are unrelated to the change you are attempting to merge in.

**Comment by voellm** [comment]
Thanks for creating this!  You can see the community is pretty fast on code reviews.  To take the code we will need the CLA signed.  It basically states the code is yours and you are free to share it.  You can read more here - https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/CONTRIBUTING.md

**Comment by lwatta** [comment]
I signed it

**Comment by lwatta** [comment]
I signed it
 

    On Wednesday, January 6, 2016 2:27 PM, googlebot <notifications@github.com> wrote:
 

 We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for the commit author(s). If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA (login here to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.—

**Comment by skschneider** [comment]
@googlebot is probably having troubles with the commits from @hildrum, similar to when she first tried to merge them in #661. If you fix the issue about extra commits in this PR, I think it will also solve the @googlebot complaint.

**Comment by voellm** [comment]
@lwatta we can also looking you up manually if you send me voellm@google.com
the name + email you signed the CLA with.

On Wed, Jan 6, 2016 at 3:11 PM, skschneider <notifications@github.com>
wrote:

> @googlebot <https://github.com/googlebot> is probably having troubles
> with the commits from @hildrum <https://github.com/hildrum>, similar to
> when she first tried to merge them in #661
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/661>. If
> you fix the issue about extra commits in this PR, I think it will also
> solve the @googlebot <https://github.com/googlebot> complaint.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/783#issuecomment-169491481>
> .
>



-- 
Anthony F. Voellm (aka Tony)
Google Voice:  (650) 516-7382
<https://www.google.com/voice/b/0?pli=1#phones>
Blog: http://perfguy.blogspot.com/
Benchmark like a pro... PerfKitBenchmarker
<https://github.com/GoogleCloudPlatform/PerfKitBenchmarker>

**Comment by lwatta** [comment]
How do I rebase around @hildrum?

**Comment by cmccoy** [comment]
You can try:

```
git rebase -i origin/master
```

In the editor window that appears, delete the two lines before your commit (`Fix proposed enhancement #780`)

If you have any trouble, `git rebase --abort` or `git reset --hard 17b6c70` should get you back to where you started.

**Comment by lwatta** [comment]
I made a mistake on my side.I'm going to cancel the pull request and start over.I'm unable to rebase around the other two changes.
 

    On Wednesday, January 6, 2016 4:02 PM, Connor McCoy <notifications@github.com> wrote:
 

 You can try:git rebase -i origin/master
In the editor window that appears, delete the two lines before your commit (Fix proposed enhancement #780)If you have any trouble, git rebase --abort or git reset --hard 17b6c70 should get you back to where you started.—

**Pull request closed without merging**